src/Form/inputTypeConfig.js

```diff
@@ -23,14 +23,15 @@ const getIdentitiesIcon = (input, {device}) => {
 
 /**
  * A map of config objects. These help by centralising here some complexity
- * @type {Object<SupportedMainTypes, InputTypeConfig>}
+ * TODO: Why is `emailNew` not part of 'SupportedMainTypes'?
+ * @type {Record<SupportedMainTypes & {'emailNew': InputTypeConfig}, InputTypeConfig>}
  */
 const inputTypeConfig = {
     emailNew: {
```

Comment:
I think I had already removed this. Maybe I remember incorrectly. Anyway, this should be removed soon.